service_discovery_manager/Project/Android/.idea/codeStyles/Project.xml

```diff
@@ -1,29 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
```

Comment:
Please keep this file.

Reply:
This is removed by android studio. For now, keep current changes.